osu.Game/Database/UserLookupCache.cs

```diff
@@ -27,6 +27,30 @@ public class UserLookupCache : MemoryCachingComponent<int, User>
         [ItemCanBeNull]
         public Task<User> GetUserAsync(int userId, CancellationToken token = default) => GetAsync(userId, token);
 
+        /// <summary>
+        /// Perform an API lookup on the specified users, populating a <see cref="User"/> model.
+        /// </summary>
+        /// <param name="userIds">The users to lookup.</param>
+        /// <param name="token">An optional cancellation token.</param>
+        /// <returns>.</returns>
```

Comment:
Probably should mention that this can return `null` for users that couldn't be retrieved.

Reply:
Oh I meant to skip returning the nulls actually, got distracted (my usage of it doesn't guard against this either).

Reply:
Aha, yeah I think that would be even better.

Reply:
Turns out doing so would be harder than expected so I just added xmldoc and checks